core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -1646,6 +1646,10 @@ public String getDisplayName() {
         return jdks;
     }
 
+    public void setJDKs(Collection<? extends JDK> jdks) {
```

Comment:
Definitely restricted, especially if we consider moving JDK installer into plugin.